src/components/ShoppingCart.js

```diff
@@ -25,43 +25,60 @@ const ShoppingCart = ({
   }
 
   return (
-    <>
-      <div
-        style={{
-          position: "fixed",
-          top: 0,
-          bottom: 0,
-          right: isOpen ? 0 : -300,
-          width: 300,
-          height: "100%",
-          backgroundColor: "white",
-          border: ".5px grey",
-          transition: "right 0.3s ease-in-out",
-          zIndex: 1,
-          opacity: isOpen ? 1 : 0,
-        }}
-      >
-        <section id="cart-header">
-          <h3>Your Cart</h3>
-          <button className="cart-x-button" onClick={() => setIsOpen(!isOpen)}>
-            X
-          </button>
-        </section>
-        <div>
-          {cart.map((p) => (
-            <div key={p.id}>
-              <img width="50px" src={p.img} alt="" />
-              <h2>{p.name}</h2>
-              <p>$ {p.price}</p>
-              <button onClick={() => handleDecrementBtn(p)}>-</button>
-              <span>{p.count}</span>
-              <button onClick={() => updateCart(p)}>+</button>
-              <button onClick={() => removeProductBtn(p)}>Remove</button>
+    <div
+      style={{
+        position: "fixed",
+        top: 0,
+        bottom: 0,
+        right: isOpen ? 0 : -300,
+        width: 300,
+        height: "100%",
+        backgroundColor: "white",
+        border: ".5px grey",
+        transition: "right 0.3s ease-in-out",
+        zIndex: 1,
+        opacity: isOpen ? 1 : 0,
+      }}
+    >
+      <section id="cart-header">
+        <h3>Your Cart</h3>
+        <button className="cart-x-button" onClick={() => setIsOpen(!isOpen)}>
+          x
+        </button>
+      </section>
+      <div>
+        {cart.map((p) => (
+          <div className="cart-product-info" key={p.id}>
+            <img className="cart-img" src={p.img} alt="products-images" />
+            <div className="inside-product-info">
+              <h2 className="product-name">{p.name}</h2>
+              <span>$ {p.price}</span> <br></br>
```

Comment:
It's better to add spaces between elements with CSS rather than adding break tags `<br></br>`